packages/css/src/utils/sort-pseudo-selectors.ts

```diff
@@ -0,0 +1,22 @@
+import type { Rule } from 'postcss';
+
+import { styleOrder } from './style-ordering';
+
+export const getPseudoSelectorScore = (selector: string): number => {
+  const index = styleOrder.findIndex((pseudoClass) => selector.trim().endsWith(pseudoClass));
+  return index + 1;
+};
+
+/**
+ * Given a list of CSS rules, sort the rules in-place such that the pseudo-selectors are
+ * sorted with the ordering defined in `packages/css/src/utils/style-ordering.ts`.
+ *
+ * @param rules
+ */
+export const sortPseudoSelectors = (rules: Rule[]): void => {
```

Comment:
this was copied over from https://github.com/atlassian-labs/compiled/pull/1666 to reduce the severity of merge conflicts that arise between the two PRs